README.md

```diff
@@ -58,41 +58,72 @@ The best way to ensure long-term access to the code base is to clone or fork the
 
 To operate in trial, test or 'bare-bones' mode, scripts are offered to perform operations with no installation except Maven (with JDK as required) and `bash` as a command line environment.
 
-The utilities are however designed for integration in a range of environments, and core functionalities are implemented in XSLT 3, which is supported across platforms including Java, node JS and C.
+For more regular use or for easier integration (in some contexts), the same functionalities are provided through a `make` interface.
+
+The utilities themselves are designed for integration in a range of environments, and core functionalities are implemented in XSLT 3, which is supported across platforms including Java, node JS and C. Please deconstruct and reverse engineer. (Consider proposing improvements as [contributions](CONTRIBUTING.md).)
 
 The software is designed to be used in a range of ways:
 
 - Directly, in development of metaschemas and Metaschema-based software and tools
 - Within Metaschema-based builds, including under CI/CD, to generate artifacts or productions from metaschema source under controlled conditions
 
-### To run
+### Using `make` utility
+
+**Work in progress. Please work with us.**
+
+[`make`](https://www.gnu.org/software/make/) is helpful for providing a clean and versatile interface on the command line, with features supporting build management and process dispatching (parallelization). `make` comes pre-installed in many Linux distributions.
+
+We recommend running `make` from a bash command line under Linux or WSL, and using `make help` for discovery of its features (from any subdirectory in the project):
+
+```bash
+$ make help
+```
+
+Note that depending on the subdirectory, the help offered will be different.
+
+Run directly from script for more transparency, and see the next section for more details on available processes.
+
+### Directly from script
```

Comment:
We should mention the "super" script in `./bin/metaschema-xslt` (also mention adding it to the path)

Reply:
Just to be clear with the 👍 I think this change is important, but not blocking my approval.